datanode/sqlstore/migrations/0005_orders_pk.sql

```diff
@@ -0,0 +1,12 @@
+-- +goose Up
+
+-- CREATE TABLE orders_new AS orders;
+ALTER TABLE orders DROP CONSTRAINT IF EXISTS orders_pkey;
+
+-- ALTER TABLE orders ADD CONSTRAINT orders_pkey PRIMARY KEY (vega_time, seq_num, id);
+
+-- +goose Down
+
+-- ALTER TABLE orders DROP CONSTRAINT IF EXISTS orders_pkey;
+
+ALTER TABLE orders ADD CONSTRAINT orders_pkey PRIMARY KEY (vega_time, seq_num); -- not sure if we really can go back
```

Comment:
the PK already is already `PRIMARY KEY(vega_time, seq_num)`   why drop and recreate the same PK?